cal/cal/views.py

```diff
@@ -126,6 +126,7 @@ def google_auth(request):
         storage.put(credential)
         profile, _ = Profile.get_or_create(user=request.user)
         profile.authed = True
+        profile.first_time_authed = True
```

Comment:
Ohhh ok I was under the impression that my task was to run a sync ONLY when after a first time log in, and do nothing for all other logins. Thanks I think the fix should be pretty simple